src/igvfd/types/variant.py

```diff
@@ -0,0 +1,45 @@
+from this import d
```

Comment:
This is strange. This is printing the Zen of Python to stdout (https://peps.python.org/pep-0020/).